docs/source/data/subscriptions.mdx

```diff
@@ -3,15 +3,13 @@ title: Subscriptions in Apollo Server
 description: Persistent GraphQL read operations
 ---
 
-> **Apollo Server 3 removes built-in support for subscriptions.** You can reenable support as [described below](#enabling-subscriptions).
```

Comment:
While we probably should change how it is described, we probably should continue to open with a note that Apollo Server does not have built-in support for subscriptions.